test/org/zaproxy/zap/extension/openapi/HTTPDTestServer.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Yes, that has been bothering me! 
Its a bit tricky while we have 3 branches .. but I guess we could create some generic classes in one of them (eg master) then jar just those up and use them in the other 2 branches as a lib?